cnd/src/btsieve/ethereum.rs

```diff
@@ -178,32 +221,33 @@ where
                     })
                 });
                 if !maybe_contains_transaction {
-                    tracing::trace!(
-                        "bloom filter indicates that block does not contain transaction:
-                {:x}",
-                        block_hash,
-                    );
+                    tracing::trace!("bloom filter says no");
```

Comment:
You seem to be misunderstanding how tracing works. The span's created around these messages attach a context to every message. Have a look at the screenshots I posted and you will see that all the information is available without duplicating it in the messages.

I would strongly encourage everyone to adopt this style of writing log messages because they are much more useful and easier to write compared to including a lot of details in every single message.

Reply:
I get that and actually googled tracing::span beforehand and I agree that this is very useful. Nevertheless I disagree with this log message. 

Imho a log message should be useful while reading the log. If I would read `blockhash = XXX , bloom filter says no` I wouldn't know what that means without knowing the code. 
`bloom filter indicates that block does not contain transaction` or similar however does. 

Note: I'm not talking about the `block_hash` in the message.

Reply:
Now I get you.

I just wanted to put in a Little-Britain reference but I can change it back if you want :)

https://www.youtube.com/watch?v=0n_Ty_72Qds

Reply:
That's exactly what I was thinking of when reading the log 😅

Reply:
But yes please change it back. 
I think it is important to not have cryptic log entries which assume you have a deep understanding of the program internals. 

```suggestion
                    tracing::trace!("bloom filter indicates that block does not contain transaction.");
```

Reply:
@tcharding Can you amend the commit please to use the suggested log message?

Reply:
Nvm I will do it!